safe_transaction_service/contracts/tasks.py

```diff
@@ -23,14 +23,17 @@
 
 logger = get_task_logger(__name__)
 
+TASK_SOFT_TIME_LIMIT = 30  # 30 seconds
+TASK_TIME_LIMIT = 60  # 1 minute
+
 
 class ContractAction(Enum):
     CREATED = 0
     UPDATED = 1
     NOT_MODIFIED = 2
 
 
-@app.shared_task()
+@app.shared_task(soft_time_limit=TASK_SOFT_TIME_LIMIT, time_limit=TASK_TIME_LIMIT)
```

Comment:
Is necessary useful the soft time limit? https://docs.celeryq.dev/en/2.3-archived/userguide/workers.html#time-limits 
Without catch this exception maybe don't make sense, what do you think?

Reply:
I prefer to always try to stop the process in a "polite" way before killing it. We will be able to see in the logs the soft limit errors